### ISSUE TITLE
Can I train a model to get segmentation masks using only ground truth bounding boxes?

### ISSUE THREAD

**Issue by rishabh-shrivastava** [matterport/Mask_RCNN#1215]
Basically I am trying to remove background from my 1 object images and I do not have datasets with ground truth masks to train my model covering all my categories?

**Comment by cam4ani** [comment]
Hi! I would advise you to use http://www.robots.ox.ac.uk/~vgg/software/via/ to produce mask on some train images (trying to choose a set of most informative images). Few hours should be enough to produce 200-400 images depending on their complexity. Then train Mask Rcnn using pretrained Imagenet weights. You might have surprisingly good results with only these several hundred images (again it depend on the complexity of those, but it might worth to try).

**Comment by rishabh-shrivastava** [comment]
> Hi! I would advise you to use http://www.robots.ox.ac.uk/~vgg/software/via/ to produce mask on some train images (trying to choose a set of most informative images). Few hours should be enough to produce 200-400 images depending on their complexity. Then train Mask Rcnn using pretrained Imagenet weights. You might have surprisingly good results with only these several hundred images (again it depend on the complexity of those, but it might worth to try).

Thank you @cam4ani . I will try this approach!

**Comment by Theosau** [comment]
hello @rishabh-shrivastava ! I see this was a while ago but did this method work?

**Comment by rishabh-shrivastava** [comment]
Sorry, I couldn't try it out.
